core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -201,6 +201,19 @@ CudaHostPinnedSpace::CudaHostPinnedSpace() {}
 // <editor-fold desc="allocate()"> {{{1
 
 void *CudaSpace::allocate(const size_t arg_alloc_size) const {
+  return allocate("[unlabeled]", arg_alloc_size);
```

Comment:
this call doesn't work doesn't? I mean there doesn't seem to be an 2 argument allocate overload. Maybe arg_logical_size should just be defaulted to the arg_alloc_size thing. Or we should just report out physical allocation size instead of logical.

Reply:
So, the three-arg does have a default argument, it defaults to `0`.

Reply:
ah ok the default is not in the cpp file I guess.